WcaOnRails/Envfile

```diff
@@ -31,3 +31,7 @@ variable :RECAPTCHA_PRIVATE_KEY, :string, default: ''
 variable :NEW_RELIC_LICENSE_KEY, :string, default: ''
 
 variable :GOOGLE_MAPS_API_KEY, :string, default: 'AIzaSyCthnikDc6olZ2lftHfqb_nkRuRrqBp17w'
+variable :STRIPE_API_KEY, :string, default: 'sk_test_vEAI1zKp934C2ML0BUPHziXd'
+variable :STRIPE_PUBLISHABLE_KEY, :string, default: 'pk_test_A1Ne6QlSRbKvnL24t3gbTva0'
+variable :STRIPE_CLIENT_ID, :string, default: 'ca_9SMJGf8Xc4MhDNdmEWMRJuWBraH3tNRk'
+variable :STRIPE_REDIRECT_URI, :string, default: 'http://localhost:3000/stripe-connect'
```

Comment:
It doesn't appear so. I'll get rid of it. This is actually set in the Stripe website.